GitHeart/Users/List/UsersListViewModel.swift

```diff
@@ -91,6 +91,7 @@ class UsersListViewModel {
 
     /// Returns number of users in the list.
     func numberOfUsers() -> Int {
+      // Return is not required on one-liners
```

Comment:
It's a habit :). I need to configure SwiftFormat to do that automatically if it supports that.